src/home/views.py

```
@@ -19,3 +19,33 @@
+        if post["active"] == True:  # Dovrebbe essere 'is True'
+            result_posts.append(post["title"])
+
+    # Problema 3: String concatenation invece di f-string
+    message = "User " + user.username + " has " + str(len(result_posts)) + " posts"
+
+    # Problema 4: Doppia negazione
+    if not user.is_active != True:
+        pass
+
+    # Problema 5: Unused import e variable
+    unused_var = "test"
```

Flake8 (7.2.0)

[error] local variable 'unused_var' is assigned to but never used

(F841)